pkg/reconciler/brokercell/brokercell.go

```diff
@@ -175,7 +175,17 @@ func (r *Reconciler) ReconcileKind(ctx context.Context, bc *intv1alpha1.BrokerCe
 		bc.Status.MarkIngressFailed("IngressServiceFailed", "Failed to reconcile ingress service: %v", err)
 		return err
 	}
-	bc.Status.PropagateIngressAvailability(endpoints)
+	// If deployment has replicaUnavailable error, it potentially has authentication configuration issues.
+	if replicaAvailable := bc.Status.PropagateIngressAvailability(endpoints, ind); !replicaAvailable {
+		podList, err := authcheck.GetPodList(ctx, resources.GetLabelSelector(bc.Name, resources.IngressName), r.KubeClientSet, bc.Namespace)
+		if err != nil {
+			logging.FromContext(ctx).Error("Failed to propagate authentication check message from ingress component", zap.Any("namespace", bc.Namespace), zap.Any("name", bc.Name), zap.Error(err))
```

Comment:
```suggestion
			logging.FromContext(ctx).Error("Failed to propagate authentication check message from ingress component", zap.Error(err))
```

The name and namespace of the thing being reconciled are already in the logger, in the `knative.dev/key` field.